agent/appconfig/constants_unix.go

```diff
@@ -111,5 +111,9 @@ func init() {
 			DefaultDocumentWorker = "/snap/amazon-ssm-agent/current/ssm-document-worker"
 
 		}
+		ssmDefaultDocumentWorkerFromEnv := os.Getenv("AWS_SSM_DEFAULT_DOCUMENT_WORKER")
```

Comment:
I tried to make the envvar look official-like as much as possible.
Do you like it? 🤔